metal3-scaling-experiments/README.md

```diff
@@ -51,156 +53,155 @@ Caveats:
+cert-manager                        cert-manager-cainjector-ffb4747bb-bqws7                          2m           157Mi
+cert-manager                        cert-manager-webhook-545bd5d7d8-5b5cp                            1m           12Mi
+kube-system                         coredns-565d847f94-4bhsl                                         3m           16Mi
+kube-system                         coredns-565d847f94-zk9m2                                         4m           15Mi
+kube-system                         etcd-kind-control-plane                                          576m         241Mi
+kube-system                         etcd-kind-control-plane2                                         296m         234Mi
+kube-system                         etcd-kind-control-plane3                                         414m         227Mi
+kube-system                         kindnet-6f78x                                                    1m           8Mi
+kube-system                         kindnet-bgzxj                                                    1m           9Mi
+kube-system                         kindnet-tdxk6                                                    1m           10Mi
+kube-system                         kube-apiserver-kind-control-plane                                3559m        4418Mi
+kube-system                         kube-apiserver-kind-control-plane2                               950m         4255Mi
+kube-system                         kube-apiserver-kind-control-plane3                               1935m        4241Mi
+kube-system                         kube-controller-manager-kind-control-plane                       2m           16Mi
+kube-system                         kube-controller-manager-kind-control-plane2                      81m          343Mi
+kube-system                         kube-controller-manager-kind-control-plane3                      2m           16Mi
+kube-system                         kube-proxy-bs2th                                                 1m           12Mi
+kube-system                         kube-proxy-f2pbc                                                 1m           12Mi
+kube-system                         kube-proxy-rn5hl                                                 1m           12Mi
+kube-system                         kube-scheduler-kind-control-plane                                4m           18Mi
+kube-system                         kube-scheduler-kind-control-plane2                               4m           19Mi
+kube-system                         kube-scheduler-kind-control-plane3                               4m           21Mi
+kube-system                         metrics-server-55dd79d7bf-tcfj7                                  6m           19Mi
+local-path-storage                  local-path-provisioner-684f458cdd-dqs6c                          1m           7Mi
 ```
 
 ### CAPD scaling for comparison
```

Comment:
Would it be possible to show a side by side comparison between CAPD and CAPM3 ? I mean speed comparison
for 100 , 250, 500 and a 1000 clusters?

Reply:
The speed is not really comparable between them since they do different things. CAPD will actually spin up pods in the target clusters. And we cannot get to the same scale with CAPD "out of the box" since it requires so much resources.

It would be possible to do it with backing clusters for CAPD also I guess (remote docker socket), but I'm not sure it is useful in the current context. We just wanted to see what it takes to get to 1000 clusters with Metal3. Why I included CAPD here to begin with was to see if the experimental setup was flawed in some way as a comparison to something "real".